pkg/aave.go

```diff
@@ -106,7 +107,35 @@ type AaveOperation struct {
 	client *ethclient.Client
 }
 
+func isAaveChainSupported(chainID *big.Int, fork AaveProtocolFork) error {
```

Comment:
An opportunity to add an interface for all protocols?
```Go
type Chain interface {
    IsSupported(chainID *big.Int) bool
}
```

Reply:
not at all @blewater , Aave is unique in this sense that multiple protocols are a fork of the actual contract. Other protocols do not have any project that forked their code and is doing well enough for any one to use. So it would really be for Aave and it's forks.